--- server.xml	2020-05-13 19:09:12.000000000 +0000
+++ server.xml.nologging	2020-05-17 15:36:47.361138000 +0000
@@ -159,10 +159,6 @@
         <!-- Access log processes all example.
              Documentation at: /docs/config/valve.html
              Note: The pattern used is equivalent to using pattern="common" -->
-        <Valve className="org.apache.catalina.valves.AccessLogValve" directory="logs"
-               prefix="localhost_access_log" suffix=".txt"
-               pattern="%h %l %u %t &quot;%r&quot; %s %b" />
-
       </Host>
     </Engine>
   </Service>
