--- server.xml.turnOnAjp	2023-01-17 12:53:45.160869124 -0500
+++ server.xml.nologging	2023-01-17 12:57:49.045158802 -0500
@@ -155,9 +155,9 @@
         <!-- Access log processes all example.
              Documentation at: /docs/config/valve.html
              Note: The pattern used is equivalent to using pattern="common" -->
-        <Valve className="org.apache.catalina.valves.AccessLogValve" directory="logs"
-               prefix="localhost_access_log" suffix=".txt"
-               pattern="%h %l %u %t &quot;%r&quot; %s %b" />
+
+
+
 
       </Host>
     </Engine>
